cli/src/alluxio.org/cli/cmd/exec/test_ufsio.go

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
talked to @jiacheliu3. ufs io stress bench is not fully functional on dora. so maybe we can remove this command until it's really needed. @Xenorith

Reply:
i am fine to remove any commands that are not working. can always be readded later. this applies to all commands to be migrated, not only the ones in this PR.